src/components/forms/OfficeHoursForm.tsx

```diff
@@ -0,0 +1,397 @@
+      px={5}
+      pt={8}
+      pb={20}
+    >
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <FormControl isRequired mb={8}>
+          <FormLabel htmlFor='firstName'>
+            <PageText display='inline' fontSize='input'>
+              First Name
+            </PageText>
+          </FormLabel>
+          <Input
+            id='firstName'
+            type='text'
+            placeholder='Enter your first name'
+            bg='white'
+            borderRadius={0}
+            borderColor='brand.border'
+            h='56px'
+            _placeholder={{ fontSize: 'input' }}
+            color='brand.paragraph'
+            fontSize='input'
+          />
+        </FormControl>
+
+        <FormControl isRequired mb={8}>
+          <FormLabel htmlFor='lastName'>
+            <PageText display='inline' fontSize='input'>
+              Last Name
+            </PageText>
+          </FormLabel>
+          <Input
+            id='lastName'
+            type='text'
+            placeholder='Enter your last name'
+            bg='white'
+            borderRadius={0}
+            borderColor='brand.border'
+            h='56px'
+            _placeholder={{ fontSize: 'input' }}
+            color='brand.paragraph'
+            fontSize='input'
+          />
+        </FormControl>
+
+        <FormControl isRequired mb={8}>
+          <FormLabel htmlFor='email'>
+            <PageText display='inline' fontSize='input'>
+              Email
+            </PageText>
+          </FormLabel>
+          <Input
+            id='email'
+            type='email'
+            placeholder='Enter your Email'
+            bg='white'
+            borderRadius={0}
+            borderColor='brand.border'
+            h='56px'
+            _placeholder={{ fontSize: 'input' }}
+            color='brand.paragraph'
+            fontSize='input'
+          />
+        </FormControl>
+
+        <FormControl isRequired mb={2}>
+          <FormLabel htmlFor='individualOrTeam' mb={4}>
+            <PageText display='inline' fontSize='input'>
+              Are you submitting on behalf of a team, or as an individual?
+            </PageText>
+          </FormLabel>
+
+          <RadioGroup
+            onChange={handleRadioButton}
+            value={individualOrTeam}
+            id='individualOrTeam'
+            fontSize='input'
+            colorScheme='white'
+          >
+            <Stack direction='row'>
+              <Radio size='lg' name='individualOrTeam' value='Individual' defaultChecked mr={8}>
+                <PageText fontSize='input'>Individual</PageText>
+              </Radio>
+
+              <Radio size='lg' name='individualOrTeam' value='Team'>
+                <PageText fontSize='input'>Team</PageText>
+              </Radio>
+            </Stack>
+          </RadioGroup>
+        </FormControl>
+
+        {individualOrTeam === TEAM && (
+          <FormControl isRequired mb={8}>
```

Comment:
I think you would need the `mt={8}` here too

![image](https://user-images.githubusercontent.com/468158/150349975-43e14347-2385-4de0-a33e-43a234f3e67d.png)

Reply:
fixed!